### PR TITLE
add alternative login method

### PR DESCRIPTION
**Pull request by andesol**
## Done

Add an option, `--new` that you can pass with any command. This will trigger an alternative login flow that uses puppeteer to go through the login instead of the current implementation, which is more low level.

This is an attempt to solve authentication issues for some users.

If this works and doesn't cause any side effect, it can later become the standard behaviour.

## QA

```
yarn dev login --new
```

Try to run a command and check that it works:

```
yarn dev replicate -i --new
```

Try to run a command without `--new` and check that it runs too:

```
yarn dev replicate -i
```

## Issue

https://warthogs.atlassian.net/browse/WD-6864

### CODE REVIEW

**Comment by anthonydillon** [comment]
@andesol can this land? We are still hitting Auth issues.

**Comment by andesol** [comment]
@anthonydillon Not sure if the errors you're having are what this PR is supposed to address (this was meant for the "wrong credentials" type of error).

But in any case we can deploy to edge and try. I need a review, maybe @nathanclairmonte? :eyes:

File: src/__tests__/Job.test.ts [hunk 1]
```diff
@@ -73,7 +73,7 @@ describe("Job tests", () => {
             jest.spyOn(Job.prototype, "getBoardToPost").mockImplementation(
                 (): any => {
                     return [{ name: "Canonical - Jobs" }];
-                }
+                },
```
Comment (andesol) [inline src/__tests__/Job.test.ts:76]: This is the result of running `yarn format`

**Comment by anthonydillon** [comment]
Exactly that. We have reports (one of them) that they still can't Auth via ght.

**Review by nathanclairmonte — APPROVED** [review]
LGTM 🚀

**Pull request merged**